kajas/src/app/profile/profile.component.html

```diff
@@ -0,0 +1,73 @@
+<body>
+  <app-header></app-header>
+  <div class="profile-container">
+    <div class="container">
+      <div class="profile-picture">
+        <img
+          src="../../assets/icons/linkedin.png"
+          alt="Profile Picture"
+          class="profile-picture"
+        />
+      </div>
+      <div class="profile-info">
+        <h1>John Doe</h1>
+        <h2>Glasgow, Scotland</h2>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        </p>
+        <div class="social-links">
+          <a href="#"
+            ><img src="../../assets/icons/linkedin.png" alt="LinkedIn"
+          /></a>
+          <a href="#"
+            ><img src="../../assets/icons/facebook.png" alt="Facebook"
+          /></a>
+          <a href="#"
+            ><img src="../../assets/icons/instagram.png" alt="Instagram"
```

Comment:
### Remove unnecessary breaking of lines.
- Place line breaks in proper places.
[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_27](url)